src/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixer.php

```diff
@@ -28,6 +28,13 @@
 final class NoExtraConsecutiveBlankLinesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
     /**
+     * @var array
+     */
+    private static $defaultConfiguration = array(
```

Comment:
This field is currently not used yet, how about adding descriptions in a separate PR, then we can use it later?